doc_user/UserManual.adoc

```diff
@@ -1832,12 +1832,13 @@ One can hide the thumbnails on the left by changing the _DICOMTHUMBNAILS_ flag i
 [#therapy-management]
 === 10.10 Therapy Management ([.underline]##T##herapy)
 
-To manage the therapy of a patient highlight the patient in the *_Patient Browser_* window. Next,
+To manage the therapy of a patient, select the patient in the *_Patient Browser_* window. Next,
 press the *[.underline]##T##herapy* button. The specific window for therapy management of the selected patient appears. The window has no name but it is typically referred to as the *_Therapy_* window.
+The name of the selected patient is shown at the top.
 
-In the *_Therapy_* window insertion, modification, or removal of a therapy is possible.
+In the *_Therapy_* window, it is possible to insert, modify or remove a therapy for the selected patient.
 
-In addition from the *_Therapy_* window insertion or removal scheduled visits is accomplished.
+In addition, from the *_Therapy_* window, it is possible to insert or remove (no modify) scheduled visits for the selected patient.
```

Comment:
should `(no modify)` be `(not modify)` ?

Reply:
yes, thanks!